src/axolotl/loaders/model.py

```
@@ -390,6 +405,86 @@ def _apply_post_lora_load_setup(self, skip_move_to_device: bool):
+        remaining_world_size = world_size
+
+        if tensor_parallel_size and tensor_parallel_size > 1:
+            pc_kwargs["tp_size"] = tensor_parallel_size
+            remaining_world_size = remaining_world_size // tensor_parallel_size
+
+        if context_parallel_size and context_parallel_size > 1:
+            pc_kwargs["cp_size"] = context_parallel_size
+            remaining_world_size = remaining_world_size // context_parallel_size
+
+        if dp_shard_size is None and dp_replicate_size in (None, 1):
+            if remaining_world_size > 1:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)